Intersect.Server/Web/RestApi/Types/Player/ClassChange.cs

```diff
@@ -0,0 +1,6 @@
+namespace Intersect.Server.Web.RestApi.Types.Player;
+
+public partial struct ClassChange
```

Comment:
This is what I meant about consolidating with the SpellInfo class, the actual contents of this are so generic they can be shared be tween the two